src/components/MyAccount/SettingsTab/PasswordChangeForm.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react"
+import {
+  Form,
+  FormField,
+  TextInput,
+  Button,
+} from "@nypl/design-system-react-components"
+import styles from "../../../../styles/components/MyAccount.module.scss"
+import { BASE_URL } from "../../../config/constants"
+import type { Patron } from "../../../types/myAccountTypes"
+
+const PasswordChangeForm = ({
+  patron,
+  updateModal,
+}: {
+  patron: Patron
+  updateModal: (errorMessage?: string) => void
+}) => {
+  const [formData, setFormData] = useState({
+    oldPassword: "",
+    newPassword: "",
+    confirmPassword: "",
+    passwordsMatch: true,
+  })
+
+  const validateForm = () => {
```

Comment:
Because of how the component re-renders, you can make this a boolean instead of a function and it will work just the same.